android/libraries/rib-base/src/main/java/com/badoo/ribs/core/routing/configuration/feature/OngoingTransition.kt

```diff
@@ -0,0 +1,75 @@
+package com.badoo.ribs.core.routing.configuration.feature
+
+import android.os.Handler
+import android.os.Parcelable
+import com.badoo.ribs.core.routing.configuration.action.single.Action
+import com.badoo.ribs.core.routing.transition.TransitionDirection
+import com.badoo.ribs.core.routing.transition.TransitionElement
+import com.badoo.ribs.core.routing.transition.TransitionPair
+import io.reactivex.ObservableEmitter
+
+internal class OngoingTransition<C : Parcelable>(
+    val descriptor: TransitionDescriptor,
+    val direction: TransitionDirection,
+    private val transitionPair: TransitionPair,
+    private val actions: List<Action<C>>,
+    private val transitionElements: List<TransitionElement<C>>,
+    private val emitter: ObservableEmitter<List<ConfigurationFeature.Effect<C>>>
```

Comment:
Maybe introduce a typealias for this?
Also why list if you `flatMapIterable` it later anyway?